code/game/objects/items/shuttle_creator.dm

```diff
@@ -0,0 +1,299 @@
+#define SHUTTLE_CREATOR_MAX_SIZE 200
+#define CUSTOM_SHUTTLE_LIMIT 4
```

Comment:
Possible to have these as config options? Would be nice to let server hosts choose how many custom shuttles can be created to scale with their hardware

Reply:
Will do

Reply:
(gunna make all the changes tommorow after the test and when I got more time)